### PR TITLE
Refactored monsterframes

### PR DESCRIPTION
**Pull request by mc-crafty**
I've refactored the monster frame so all icons are standard and will load consistently. This was done by using code from addon.ipf/mobinfobyskill/mobinfobyskill.lua as noted in the changed file.
I've added the movement type to the frame.
I've also added some conditional padding so that the info displays correctly when an elite monster is targeted. They have different frames, so the spacing works out odd.
I've left the text portion of the size commented in case people prefer that over the icons. I like the icons myself.
I've also left the exp part commented. Still can't get it to work right when an elite monster shares a name with a normal monster.

### CODE REVIEW

File: monsterframes.lua [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+    if msg == 'TARGET_UPDATE' then
+        local stat = info.GetStat(session.GetTargetHandle());
+        if stat == nil then
+            return;
+        end
+
+        local targetinfo = info.GetTargetInfo(session.GetTargetHandle());
+
+        local eSpaceV = 0;
```
Comment (Excrulon) [inline monsterframes.lua]: What does the e mean? I prefer variable names fully spelled out just so anyone looking knows instantly. Same with V/H.

File: monsterframes.lua [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        numhp:SetTextAlign("center", "center");
+        numhp:SetText(ADD_THOUSANDS_SEPARATOR(stat.HP) .. "/" .. ADD_THOUSANDS_SEPARATOR(stat.maxHP));
+        numhp:SetFontName("white_16_ol");
+    end
+end
+
+--[[
+This was taken from addon.ipf/mobinfobyskill/mobinfobyskill.lua
+--]]
+function GET_MON_PROPICON_BY_PROPNAME(paramname, monclass)
```
Comment (Excrulon) [inline monsterframes.lua]: Yeah I'm familiar with this, just never got around to it. You don't have to copy and paste the code into this file as you can just use the function as is. Plus, you have outdated code. In the latest script imc has, there's another section for EffectiveAtkType which I'll add as well.

**Comment by Excrulon** [comment]
Finally got around to committing the latest code. If you can pull that down and make these changes against that file (and delete the GET_MON_PROPICON_BY_PROPNAME method), I'll pull this in.

**Comment by mc-crafty** [comment]
Kay, I've rebased. The method has been removed and the names have been expanded. It's a bit hacky to keep track if the last mob was elite or not, but it works and fast.

**Pull request merged**